pyaedt/generic/plot.py

```diff
@@ -598,10 +637,13 @@ def plot_contour(qty_to_plot, x, y, size=(2000, 1600), xlabel="", ylabel="", tit
         Color map levels. Default is `64`.
     snapshot_path : str, optional
         Full path to image to save. Default is None.
+    show : Bool, optional
+        Set to ``True`` if the figure should be rendered.
+        Default is ``True``.
```

Comment:
```suggestion
        Color map levels. The default is ``64``.
    snapshot_path : str, optional
        Full path to save the image save. The default is ``None``.
    show : bool, optional
        Whether to render the figure. The default is ``True``. If
        ``False``, the image is not drawn.
```
The edits were to make the descriptions consistent and follow stye guidelines. I feel like we shouldn't switch from figure to image unless there is some reason to. The key is to understanding is the use of consistent terms.

Reply:
@PipKat  in this context "figure" refers to a Python object while "image" refers to the rendering of that figure.